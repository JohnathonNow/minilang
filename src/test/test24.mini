fun test(N) do
	switch N: integer
	case 1 do "One"
	case 20 do "Twenty"
	case 3 .. 6 do "Few"
	case 7, 8 do "Several"
	else "Other"
	end
end

for I in 0 .. 25 do
	print('{I} -> {test(I)}\n')
end