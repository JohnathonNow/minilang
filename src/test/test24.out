0 -> Other
1 -> One
2 -> Other
3 -> Few
4 -> Few
5 -> Few
6 -> Few
7 -> Several
8 -> Several
9 -> Other
10 -> Other
11 -> Other
12 -> Other
13 -> Other
14 -> Other
15 -> Other
16 -> Other
17 -> Other
18 -> Other
19 -> Other
20 -> Twenty
21 -> Other
22 -> Other
23 -> Other
24 -> Other
25 -> Other
